source/Mole/Edge.class.st

```diff
@@ -10,32 +10,41 @@ Class {
 	#category : #Mole
 }
 
+{ #category : #private }
+Edge >> endpoints [
+
+	self subclassResponsibility
+]
+
 { #category : #testing }
 Edge >> isDirected [
 
-	^ self subclassResponsibility
```

Comment:
I keep the `^` in the subclassResponsibility when the intention is that the method doesn't return self.